src/main/java/io/github/fusionflux/portalcubed/content/PortalCubedBlocks.java

```diff
@@ -849,12 +842,22 @@ public class PortalCubedBlocks {
 			)
 			.build();
 
-	public static final PropBarrierBlock PROP_BARRIER = REGISTRAR.blocks.create("prop_barrier", PropBarrierBlock::new)
-			.copyFrom(Blocks.BARRIER)
+	public static final PropBarrierBlock PROP_BARRIER = REGISTRAR.blocks.createFrom("prop_barrier", PropBarrierBlock::new, Blocks.BARRIER)
 			.settings(QuiltBlockSettings::dynamicShape)
 			.item((block, properties) -> new BlockItem(block, properties.rarity(Rarity.EPIC)))
 			.build();
 
+	public static final Block GOO = REGISTRAR.blocks.createFrom("toxic_goo", GooBlock::new, Blocks.WATER)
+			.settings(s -> s.mapColor(MapColor.TERRACOTTA_GREEN))
+			.item(BlockItemProvider::noItem)
+			.build();
+
+	public static final Block GOO_CAULDRON = REGISTRAR.blocks.createFrom("toxic_goo_cauldron", GooCauldronBlock::new, Blocks.CAULDRON)
+			.item(BlockItemProvider::noItem)
+			.build();
+
 	public static void init() {
+		Item.BY_BLOCK.put(GOO_CAULDRON, Items.CAULDRON);
```

Comment:
???
if this is for pickblock, just override the method in the goo cauldron class

Reply:
It is for pickblock, mojang did it this way too see `Items.registerBlock(Block block, Block... blocks)` and `Items.CAULDRON`. I guess I could just override the method though